pom.xml

```diff
@@ -28,7 +28,7 @@
         <file.encoding>UTF-8</file.encoding>
         <javadoc.doclint.none>-Xdoclint:none</javadoc.doclint.none>
         <dropwizard.version>1.2.3</dropwizard.version>
-        <brave.version>4.14.1</brave.version>
+        <brave.version>4.14.3</brave.version>
         <zipkin.version>2.4.5</zipkin.version>
```

Comment:
we can probably remove this property as well since it's no longer being used.

Reply:
good catch. done